verification/c/suite/test_compiles.cpp

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Sanity tests.
+ */
+#include "gmock/gmock.h"
+#include "uavcan/time/TimeSystem_0_1.h"
+#include "uavcan/primitive/Empty_1_0.h"
+#include "uavcan/_register/Value_1_0.h"
+#include "uavcan/primitive/String_1_0.h"
+#include "uavcan/_register/Access_1_0.h"
+
+/**
+ * Verify that a simple struct type compiles.
+ */
+TEST(GeneralTests, StructCompiles) {
```

Comment:
```suggestion
TEST(GeneralTests, StructCompiles)
{
```

I think we could use clang-format here